lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -206,7 +206,21 @@ for node in $allocated_nodes; do
 done
 
 # Make sure all openstack services are down on all nodes before we stop rabbitmq
-sleep 10
+services_stopped=""
+count=0
+while [ -z "$services_stopped" -a "$count" -lt 100 ]; do
+  sleep 1
+  services_stopped="yes"
+  for node in $allocated_nodes; do
+    if ssh "$node" 'shopt -s nullglob; for i in /etc/init.d/openstack-* ; do if $i status >/dev/null ; then echo "$i still running at `hostname`"; exit 0; fi; done; exit 1'; then
```

Comment:
Oh, right. Somehow I was surprised by the logic; I guess I would have done the tests in the opposite way, but doesn't matter.